### PR TITLE
Added a definite integral calculator 

### PR DESCRIPTION
**Pull request by Devanshu53**
# Fixes Issue🛠️
closes : #92

Closes #

# Description👨‍💻 

A simple web-based definite integral calculator that allows users to calculate definite integrals of mathematical functions.
 Features

- Calculate definite integrals of various mathematical functions.
- User-friendly interface with input fields for function, lower limit, and upper limit.
- Supports a variety of mathematical functions, including polynomials, trigonometric functions, and more.
- Real-time calculation and display of results.
- Responsive design for optimal user experience on different devices.
# Type of change📄

- [] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I am an Open Source Contributor

# Screenshots/GIF📷
Definite-Integra-calculator

### CODE REVIEW

**Comment by Rakesh9100** [comment]
Resolve the conflicts pls @Devanshu53 

Delete the `.vscode` folder and your png image file from `assets/images`.

**Comment by Devanshu53** [comment]
@Rakesh9100  then how can i add the screenshot of my website in the readme file

**Comment by Rakesh9100** [comment]
Just keep your image from Desktop, copy and paste it on the Readme @Devanshu53

**Comment by Rakesh9100** [comment]
Resolve the conflicts @Devanshu53

**Comment by Rakesh9100** [comment]
Do resolve the conflicts @Devanshu53

**Comment by Devanshu53** [comment]
Please specify the Conflicts @Rakesh9100

File: index.html [hunk 1]
```diff
@@ -249,6 +249,18 @@ <h3>Calculates the Pressure conversion between two specified units.</h3>
                 </a>
             </div>
         </div>
+        <div class="box">
+            <div class="content">
+                <h2>Definite Integral Calculator</h2>
+                <h3>Evaluates Integral of Mathematical Functions with Definite Limits.</h3>
+                <a href="./Calculators/Definite-Integral-Calculator/index.html" target="_blank">
```
Comment (Rakesh9100) [inline index.html:256]: Change this `index.html` to your file name

**Comment by Rakesh9100** [comment]
Good, Merged!!  Congrats on getting your PR merged to this repo under SWOC S4 🎉🎉 @Devanshu53

**Pull request merged**